src/BenchmarkDotNet/Toolchains/InProcess.NoEmit/InProcessNoEmitExecutor.cs

```diff
@@ -127,8 +127,10 @@ private ExecuteResult GetExecutionResult(RunResults runResults, int exitCode, IL
             }
 
             var lines = runResults.GetMeasurements().Select(measurement => measurement.ToOutputLine()).ToList();
-            lines.Add(runResults.GCStats.ToOutputLine());
-            lines.Add(runResults.GCStats.ToOutputLine());
```

Comment:
There were GCStats twice. I assume that it was a mistake?

Reply:
it was most probably a bug